analyses/0014-vitess/analysis.md

```diff
@@ -0,0 +1,1019 @@
+
+**Getting started next steps**: Do users know where to go after reading the
+getting started guide?
+
+Yes, there is a Next Steps section at the end of each install except the Docker
+local install. However, the Next Steps section is perfunctory, pointing in all
+cases to the Move Tables page in the Migration guide.
+
+**New user signpost**: Is your new user content clearly signposted on your site’s
+homepage or at the top of your information architecture?
+
+No, there is no clear entry path for new users. They'll probably make their way through
+the install and then try to figure out how to implement their migration case (which
+probably fits one of the documented scenarios, but there is no clear way for them
+to find it).
+
+**Sample code**: Is there easily copy-pastable sample code or other example content?
+
+Yes, command-line examples are plentiful. There is no click-to-copy, however. Users
+must highlight and copy manually.
+
+#### Content maintainability & site mechanics
+
+**Searchability**: Is your documentation searchable?
+
+Yes, there is a text Search feature that is limited to the currently displayed version.
+It seems to work very well.
+
+Search is full-text, so common search terms can result in an unwieldy number of results.
+
+**Localization & i18n directories**: Are you planning for localization/internationalization
+with regard to site directory structure?
+
+Yes, there are full versions of the documentation in both English and Chinese.
+
+**Localization framework**: Is a localization framework present?
+
+Yes, there seems to be some infrastructure for multiple languages. I'm not sure how
+translation is done in Hugo.
+
+**Versioning**: Do you have a clearly documented method for versioning your content?
+
+Yes, releasing a new version is scripted. The repository contains instructions. The
+latest three released versions are available on the website.
+
+#### Content creation processes
+
+**Content creation process**: Is there a clearly documented (ongoing) contribution
+process for documentation?
+
+There are instructions for building and releasing the documentation, as well as detailed
+instructions for contributing instructional videos. However, there are no instructions
+for contributing technical documentation or fixing documentation issues.
+
+Presumably a contributor can submit a pull request on the website repo to amend or
+add to the documentation, but there is no procedure documented. There is a notice
+that no grammar or typo fixes are accepted from accounts less than a year old.
+
+**Release process**: Does your code release process account for documentation creation
+& updates?
+
+No, neither the [Contribute](https://vitess.io/docs/contributing/) documentation nor
+the [CONTRIBUTING.md](https://github.com/vitessio/vitess/blob/main/CONTRIBUTING.md)
+file in the product repo describes how to contribute documentation. There is no Contribution
+section in the website repo.
```

Comment:
We have this tool: https://github.com/vitessio/vitess-releaser, which maintainers use to create and ship new major and patch releases of Vitess. This tool lists and automated steps needed for a release, one of them is related to documentation and guides the maintainer on how to create and update the documentation before and after each release:
- https://github.com/vitessio/vitess-releaser/blob/main/go/releaser/release/website_docs.go

Reply:
@frouioui Does the tool verify that you've updated the release notes?